### ISSUE TITLE
Simulate a network taking into account comercial pipes (material, DN, PN,..)

### ISSUE THREAD

**Issue by jalabanda** [neslerel/QGISRed#74]
Hello,
I find the QGisRed plugin very useful and it interests me a lot. However I miss one thing.
I am dedicated to the management of irrigation networks for irrigation communities and these have pipes of different materials (cast iron, pvc, pvc-o, pe, ...) and nominal pressures.
The plugin calculates based on the diameter (inside). For me it would be very useful if you could select the type of pipe, nominal diameter and nominal pressure (I mean, comercial pipes). Above all, being able to simulate a network taking into account the nominal pressure of the pipe is essential.
I can't find if the plugin is capable of doing it, or if you have it in mind in the future.
At the moment the only thing I can think of is to model it in Epanet and import it, but I already have to walk with two different software for the same purpose.
Thanks a lot.

**Comment by fmartine** [comment]
Indeed, differentiating the inside diameter from the nominal diameter, and considering the nominal pressure as one more property of the pipes, is something that is planned to be incorporated into QGISRed at some time, but for now it is not the priority. However, if there are more users than you asking for that we can advance this improvement for the next versions.

Thank you for your comments.

**Comment by jalabanda** [comment]
Perfect then. 
I look forward to it.
Thanks so much.
